ddl/ddl_api.go

```diff
@@ -1996,6 +1991,17 @@ func resolveAlterTableSpec(ctx sessionctx.Context, specs []*ast.AlterTableSpec)
 	return validSpecs, nil
 }
 
+func isSameTypeMultiSpecs(specs []*ast.AlterTableSpec) bool {
+	specType := specs[0].Tp
+	var count int = 0
+	for _, spec := range specs {
+		if spec.Tp == specType {
+			count++
+		}
```

Comment:
```suggestion
		}
		return false
```

Reply:
Hi, you means that we can return false earlier when `spec.Tp != specType`, right?
I think the change is :
```
	specType := specs[0].Tp
	for _, spec := range specs {
		if spec.Tp != specType {
			return false
		}
	}
	return true
```

Reply:
Yes.